agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/ConfigurationTest.java

```diff
@@ -337,6 +337,16 @@ public void shouldOverrideSelfDiagnosticsLevel() throws IOException {
         assertEquals("DEBUG", configuration.selfDiagnostics.level);
     }
 
+    @Test
+    public void shouldOverrideSelfDiagnosticsFilePath() throws IOException {
+        envVars.set("APPLICATIONINSIGHTS_SELF_DIAGNOSTICS_FILE_PATH", "/tmp/ai.log");
```

Comment:
before overriding, can we assertEquals the value from the applicationinsights.json?

Reply:
the `shouldParse` test above does this